requirements.txt

```diff
@@ -4,9 +4,10 @@
 # external requirements
 awscli
 boto3==1.9.250
+black==19.3b0
```

Comment:
oh, do we really need to depend on a very specific version?

Also, packages which are needed for the CI only, and not development, shouldn't be here I think. I guess `coverage` is an example? or are we using it?